test/test_integration_configurations.py

```diff
@@ -166,10 +181,11 @@ def test_https_cli_fails_if_cert_is_bad(self):
         conjurrc = ConfigFile(account=self.client_params.account, conjur_url=self.client_params.hostname,
                               cert_file=self.environment.path_provider.nginx_conf_path)
         conjurrc.dump_to_file()
-        with open(f"{DEFAULT_NETRC_FILE}", "w") as netrc_test:
-            netrc_test.write(f"machine {self.client_params.hostname}\n")
-            netrc_test.write("login admin\n")
-            netrc_test.write(f"password {self.client_params.env_api_key}\n")
+        cred_store,_ = CredentialStoreFactory().create_credential_store()
```

Comment:
seems to be that this can be part of the test_helpers.py as `is_credentials_exist` and `get_credentials`

Reply:
`cred_store,_` -> `cred_store, _`